Common/ISubTask.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Common
```

Comment:
The name of the file is incorrect :slightly_frowning_face: It should be `ISubtask`